cockroachcloud/migrations-page.md

```diff
@@ -0,0 +1,159 @@
+
+{% comment %}
+For an example, see [Migrate a PostgreSQL database](migrate-from-postgres.html#tk).
+{% endcomment %}
+
+{{site.data.alerts.callout_info}}
+The dump file must be less than 4 MB. `INSERT` and `COPY` statements will be ignored in schema conversion.
+{{site.data.alerts.end}}
+
+To begin a database migration:
+
+1. Click the upload box and select a `.sql` file, or drop a `.sql` file directly into the box. 
+1. Wait for the schema to be analyzed. A loading screen is displayed. Depending on the size and complexity of the SQL dump, analyzing the schema can require up to several minutes.
+1. When analysis is complete, review the [**Summary Report**](#summary-report) and edit, add, or remove SQL statements in the [**Statements** list]#statements-list).
+
+## Migrations table
+
+If you have attempted at least one migration, the Migrations table is displayed with the following details:
+
+|     Column     |                                                                                         Description                                                                                          |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Migration Name | The filename of the `.sql` file that was [uploaded](#upload-a-sql-dump).                                                                                                                       |
+| Status         | The status of the migration: `READY FOR REVIEW`, `READY TO FINALIZE`, or `FINALIZED`. Migrations with `READY TO FINALIZE` status can be [finalized](#finalize-the-schema).                    |
+| Date Imported  | The timestamp when the SQL dump was uploaded.                                                                                                                                                |
+| Last Updated   | The timestamp when the [SQL statements](#statements-list) were updated.                                                                                                                      |
+| Errors         | The number of SQL errors preventing a migration from attaining `READY TO FINALIZE` status. |
+
+To view the [**Summary Report**](#summary-report) or [**Statements** list](#statements-list) for a migration, click the migration name.
+
+## Summary Report
+
+The **Summary Report** displays the results of the schema analysis:
+
+- The number of **statements total** that were analyzed in the `.sql` file that you [uploaded](#upload-a-sql-dump).
```

Comment:
This first bullet is a little confusing to me. Does it mean the total number of statements that were analyzed from the .sql file? Or the the number of totals? I assume the report displays **statements total**, so if the former it would be hard to reword this. I just wanted to point out I paused here and re-read a few times.

Reply:
I actually don't understand the second statement ("the number of totals") as I'm probably too close to this material. Does rewording it this way resolve any ambiguity?

```
- The number of **statements total** in the [uploaded](#upload-a-sql-dump) `.sql` file that were analyzed.
```